task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +97,10 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if(a+b>c && a+c>b && b+c>a){
```

Comment:
just return boolean expression. no need for if(<bolean expression>) return true else return false. Fix it for other methods as well